health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/accessbilityPopUP.js

```
@@ -62,76 +70,90 @@ const AccessibilityPopUp = ({ onClose, census }) => {
       {
         body: reqPayload,
 
-      });
+      },
+      {
+        onSuccess: (data) => {
+          onSuccess && onSuccess(data); // Call the onSuccess callback if provided
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)